tests/installation/first_boot.pm

```diff
@@ -30,7 +30,7 @@ sub run {
         $self->wait_boot(textmode => 1);
         select_console('x11');
     }
-    my $boot_timeout = (get_var('SES5_DEPLOY') || check_var('VIRSH_VMM_FAMILY', 'hyperv')) ? 450 : 200;
+    my $boot_timeout = (get_var('SES5_DEPLOY') || check_var('VIRSH_VMM_FAMILY', 'hyperv')) ? 450 : 300;
```

Comment:
I am not sure we want to go ahead and accept a 5 minute bootup of a clean system installation. If you are not sure what caused the bootup delay you can consult with @OleksandrOrlov which has experience with outputting the status of the system boot/shutdown details to find out

Reply:
@okurz, @zluo, unfortunately, I never investigated boot up problems, but if the issue is caused by too slow shut off on system reboot, then I can advise improving the [existing approach](https://github.com/os-autoinst/os-autoinst-distri-opensuse/pull/5314) to also cover the reboot case, or to try gather logs by doing the same manually at least to debug it faster right now (e.g. as described here https://freedesktop.org/wiki/Software/systemd/Debugging/#index2h1).